templates/form_theme_admin.html.twig

```diff
@@ -166,6 +166,34 @@
     </div>
 {% endblock %}
 
+{% block app_admin_elected_representative_sponsorships_sonata_type_native_collection_sponsorships_widget -%}
+    <div class="row">
+        <div class="col-xs-11 col-xs-offset-1">
+            <div class="row bg-primary">
+                <div class="col-md-2">Élection présidentielle</div>
+                <div class="col-md-10">Candidat</div>
+            </div>
+        </div>
+    </div>
+    ​
+    <div>
+        {{ form_widget(form) }}
+    </div>
+{% endblock %}
+
+{% block app_admin_elected_representative_sponsorship_sponsorships_sponsorships_entry_widget -%}
+    <div class="row" style="margin-top: 10px; margin-left: -30px;">
+        <div class="col-md-2">
+            {{ form_widget(form.presidentialElectionYear) }}
```

Comment:
why do you not use `form_row` instead of two calls `form_widget` and `form_errors`? 
if it's for avoiding form_label call, you can do it with `label: false` parameter